pkg/model/gcemodel/api_loadbalancer.go

```diff
@@ -99,6 +99,18 @@ func createPublicLB(b *APILoadBalancerBuilder, c *fi.CloudupModelBuilderContext)
 			TargetTags:   []string{b.GCETagForRole(kops.InstanceGroupRoleControlPlane)},
 			Allowed:      []string{"tcp:" + strconv.Itoa(wellknownports.KubeAPIServer)},
 		})
+
+		if b.NetworkingIsIPAlias() {
+			c.AddTask(&gcetasks.FirewallRule{
+				Name:         s(b.NameForFirewallRule("pod-cidrs-to-https-api")),
+				Lifecycle:    b.Lifecycle,
+				Network:      network,
+				SourceRanges: []string{b.Cluster.Spec.Networking.PodCIDR},
+				TargetTags:   []string{b.GCETagForRole(kops.InstanceGroupRoleControlPlane)},
+				Allowed:      []string{"tcp:" + strconv.Itoa(wellknownports.KubeAPIServer)},
+			})
+		}
+
 		if b.Cluster.UsesNoneDNS() {
 			b.AddFirewallRulesTasks(c, "kops-controller", &gcetasks.FirewallRule{
```

Comment:
Unrelated to this PR, but why does all of `b.Cluster.Spec.API.Access` need to talk to kops-controller? Shouldn't this be restricted to `NetworkCIDR`?

Reply:
I believe that's right, I can send a PR, unless you want to :-)

It might be that on some platforms it's not possible to have a separate CIDR, but it is possible here :-)

Reply:
Actually, in most cases, this will have to be `0.0.0.0/0`. With most clouds, there is no way to track the internal IP through an external load balancer.